package.json

```diff
@@ -4,46 +4,34 @@
   "version": "0.0.0",
   "description": "Minimalistic browser-based Solidity IDE",
   "scripts": {
+    "prepublish": "npm-run-all -ls downloadsolc make-mock-compiler build",
+    "build": "mkdirp build; browserify src/index.js -t yo-yoify -t [ babelify --presets [ es2015 ] ] | babili > build/app.js",
+    "start": "npm-run-all -lpr serve watch onchange",
+    "test": "standard; babel src --out-dir babelify-src; node test/index.js",
+    "selenium": "selenium-standalone start",
     "browser-test": "nightwatch --config nightwatch.js --env local",
     "browser-test-remote-chrome": "nightwatch --config nightwatch.js --env chrome",
     "browser-test-remote-firefox": "nightwatch --config nightwatch.js --env default",
     "browser-test-remote-ie": "nightwatch --config nightwatch.js --env ie",
     "browser-test-remote-safari": "nightwatch --config nightwatch.js --env safari",
     "browser-test-remote-parallel": "nightwatch --config nightwatch.js --env safari,ie,default,chrome",
-    "build": "mkdir build; browserify src/index.js -g yo-yoify -o build/app.js -t [ babelify ]; babel --plugins=transform-es2015-template-literals build/app.js --out-file build/app.js",
-    "csslint": "csslint --ignore=order-alphabetical --errors='errors,duplicate-properties,empty-rules' --exclude-list='assets/css/font-awesome.min.css' assets/css/",
```

Comment:
ok, i'll put it to alphabetic order then. didnt realize - tried to sort it according to the order its mentioned in the scripts